src/scripts/boot.js

```diff
@@ -92,7 +97,11 @@ export const startApp = async (
     routes
   })
 
+  setGoogleAnalyticsPage(router.currentRoute.path)
   router.beforeEach(routeGuard(store))
+  router.afterEach(to => {
+    setGoogleAnalyticsPage(to.path)
```

Comment:
if you don't want to test this, please add a `/* istanbul ignore next: <REASON> */`

Reply:
it would be good to test it - but i couldn't figure out how to mock the router in in app.spec - odd setup there. do you know how?

Reply:
there are actually tests for the routeGuard in `App.spec` which could be used as a reference